handlers/main.yml

```diff
@@ -18,7 +18,7 @@
   command: "{{ nginx_binary_name }} -t"
   register: result
   changed_when: "result.rc != 0"
-  always_run: yes
+  check_mode: no
```

Comment:
`check_mode` was introduced in ansible 2.2 (the last version), it make the role incompatible in previous version, it was decided in #140 to wait until Ansible 2.3 is available.